src/query/range_query_ip_fastfield.rs

```diff
@@ -139,29 +152,33 @@ struct IpRangeDocSet {
     /// Current batch of loaded docs.
     loaded_docs: VecCursor,
     last_seek_pos_opt: Option<u32>,
+    /// If fast field is multivalue.
+    is_multivalue: bool,
 }
 
-const DEFALT_FETCH_HORIZON: u32 = 128;
+const DEFAULT_FETCH_HORIZON: u32 = 128;
 impl IpRangeDocSet {
     fn new(
         value_range: RangeInclusive<Ipv6Addr>,
         ip_addr_fast_field: Arc<dyn Column<Ipv6Addr>>,
```

Comment:
For the range query we only need 2 methods:
`get_docids_for_value_range` and `num_docs`
The effective horizon is different for multivalue fastfields after translating, but that's a small tradeoff for the same code. 

I'm not sure about splitting the trait, it could be a good idea, but could also be a hassle to work with different traits.
I wouldn't do it in this PR though.